sklearn/cluster/hierarchical.py

```diff
@@ -642,9 +642,7 @@ class AgglomerativeClustering(BaseEstimator, ClusterMixin):
           all observations of the two sets.
 
     pooling_func : callable, default=np.mean
-        This combines the values of agglomerated features into a single
-        value, and should accept an array of shape [M, N] and the keyword
-        argument ``axis=1``, and reduce it to an array of size [M].
+        ignored, Deprecated to be removed in 0.22.
```

Comment:
I was not saying that you could or should not use `.. deprecated` but that leaving in an untrue description of behaviour helped no one.